Mage/src/main/java/mage/cards/repository/CardCriteria.java

```diff
@@ -33,6 +33,12 @@ public class CardCriteria {
     private Boolean doubleFaced;
     private Boolean modalDoubleFaced;
     private boolean nightCard;
+    /**
+     * If true,  only meld cards (the back face of a meld pair), return nothing if nightCard = false
+     * If false, no meld cards whatsoever.
+     * If null,  we want meld cards and not meld cards, even when nightCard = true.
```

Comment:
> we want meld cards and not meld cards

Maybe typo in the comments. Anyway -- see my comment below -- no needs in additional conditionals. Use simple meld/night (e.g. verify code uses all night cards for check, but images download split it to meld and non meld).

Reply:
Comment was not clear, but the idea was to allow meldCards to be there even when other nightcards were filtered out, but will not use that logic in final version.

I'm adding null support for nightCard instead, to not have the nightCard=false or nightCard=true on retrieving cards to download. I want both, then regular nightCards will be ignored.